### PR TITLE
Feature all messages

### PR DESCRIPTION
**Pull request by dustymeyers**
This feature renders all messages. Currently renders a list of conversations between the logged in user and other vendors. Could be bugs if logged in as a vendor atm when displaying messenger's name (currently set to companyName MessagesList.jsx:66). Able to click through to see full conversation.

### CODE REVIEW

File: src/components/MessageAll/MessageAll.jsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ function MessageAll() {
             height: 300,
             marginLeft: 10,
             marginRight: 10,
-          }}
+          }}d
```
Comment (William-Krug) [inline src/components/MessageAll/MessageAll.jsx]: I'm confused about the "d" on this line.  Typo?

**Review by William-Krug — CHANGES_REQUESTED** [review]
Please take a look at line 112 in `MessageAll.jsx`.  There are also going to be merge conflicts with this PR, please pull from Master and resolve before pushing back up.

File: src/components/MessageAll/MessageAll.jsx (hunk 1 above)
Comment (dustymeyers) [reply src/components/MessageAll/MessageAll.jsx]: Weird, I pulled everything down last night, must have been a merge that I didn't realize. I'll get that fixed nice catch^

Comment (dustymeyers) [reply src/components/MessageAll/MessageAll.jsx]: d removed and pushed up the new version with the resolved conflicts! :D

**Review by William-Krug — APPROVED** [review]
Thanks for the quick turn around

**Pull request merged**